Aliases/7kaa.rb

```diff
@@ -0,0 +1,35 @@
+class SevenKingdoms < Formula
```

Comment:
This should be a symlink, not a copy of the file

Reply:
ah I see